source/Cosmos.Debug.Kernel.Plugs.Asm/packages.lock.json

```diff
@@ -11,25 +11,27 @@
           "Microsoft.NETCore.Platforms": "1.1.0"
         }
       },
```

Comment:
Why is this dependency added?

Reply:
i think i left a using from when i was testing it